test/uglify_test.js

```diff
@@ -21,20 +21,20 @@ exports.contrib_uglify = {
       'compress_mangle_banner.js',
       'compress_mangle_beautify.js',
       'compress_mangle_except.js',
-      'enclose.js',
       'multifile.js',
       'wrap.js',
       'maxLineLen.js',
       'asciionly.js',
-      'exportAll.js',
       'screwIE8.js',
       'sourcemap_basic.js',
       'sourcemap_basic.js.map',
       'sourcemap_customDir.js',
       'sourcemap_customName.js',
+      'source_map_custom_name',
```

Comment:
The test case has been like that before this PR:
https://github.com/gruntjs/grunt-contrib-uglify/blob/master/Gruntfile.js#L176
https://github.com/gruntjs/grunt-contrib-uglify/blob/master/test/fixtures/expected/source_map_custom_name

But for some reasons this generated map file isn't being compared at all, so I thought I'd add it to the list here.